iaso/api/mobile/entity.py

```diff
@@ -63,9 +63,9 @@ class Meta:
             "json",
         ]
 
-    form_id = serializers.IntegerField(read_only=True, source="form.id")
+    form_id = serializers.IntegerField(read_only=True)
     id = serializers.CharField(read_only=True, source="uuid")
-    org_unit_id = serializers.CharField(read_only=True, source="org_unit.id")
+    org_unit_id = serializers.CharField(read_only=True)
```

Comment:
This saves us from a costly join 👍 

However this will always take a lot of time https://github.com/BLSQ/iaso/blob/f8bdac42b26f50f938f4bf46dddce4c8bc26d9a8/iaso/api/mobile/entity.py#L105